script.js

```
@@ -0,0 +1,1113 @@
+ */
+function loadTemplate(templateName) {
+    if (!codeEditorEl) {
+        console.error("Editor de código não encontrado (loadTemplate).");
+        showStatus("Erro: Editor não encontrado.", "error");
+        return;
+    }
+    if (templates.hasOwnProperty(templateName)) {
+        currentTemplate = templateName;
+        codeEditorEl.value = templates[templateName];
+
+        // Update active template visual state
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)